app/javascript/Components/App.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import ReactDOM from "react-dom";
+import { Provider } from "react-redux";
+import { BrowserRouter as Router, Routes, Route } from "react-router-dom";
+import store from "../redux/store";
+import Greeting from "./Greeting";
+
+const App = () => {
+  return (
+    <Provider store={store}>
+       <Greeting/>
+    </Provider>
+  );
+};
+
```

Comment:
- [ ] After installing the packages above your app still does not work as expected. See the error message in the console. Kindly ensure that if you have the same issue or provide a workaround.

  ![dbgft](https://user-images.githubusercontent.com/54848363/197774093-62efbc59-70c0-4b9b-a4f5-6f07126a3ce9.jpg)